### PR TITLE
Make response header more readable

### PR DESCRIPTION
**Pull request by dibericky**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: packages/api-explorer/src/components/Response/ResponseMetadata.jsx [hunk 1]
```diff
@@ -77,7 +106,20 @@ function ResponseMetadata({ result, intl }) {
...
-        {result.responseHeaders.join('\n')}
+        <div style={style.headersContainer}>
+
+          {result.responseHeaders.map(header => {
+            return (
+              <div style={style.headerRow}>
+                <span style={style.headerName}>{header.name}</span>
+                { 
+                  header.value ? <span style={style.headerValue}>{header.value}</span>
+                  : <span style={style.headerUnknownValue}>{'no-value'}</span>
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/Response/ResponseMetadata.jsx]: Use FormattedMessage instead.

**Comment by fredmaggiowski** [comment]
LGTM

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 2]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            responseBody: JSON.stringify({ a: 1 }),
+            requestBody: JSON.stringify({ b: 2 }),
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: [],
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]:
```suggestion
    const doc = wrapper.find(Response);  
```

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 3]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            requestBody: JSON.stringify({ b: 2 }),
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: [],
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');  
+    doc.instance().setTab('metadata');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]: This is necessary? Or the setState is enough?

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 4]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            responseBody: JSON.stringify({ a: 1 }),
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: [],
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]:
```suggestion
    const doc = wrapper.find(Response);  
```

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 5]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: [],
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');  
+    doc.instance().setTab('metadata');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]: As above

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 6]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            responseBody: JSON.stringify({ a: 1 }),
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: resultResponseHeaders,
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]:
```suggestion
    const doc = wrapper.find(Response);  
```

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 7]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: resultResponseHeaders,
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');  
+    doc.instance().setTab('metadata');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]: As above

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 8]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            responseBody: JSON.stringify({ a: 1 }),
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: resultResponseHeaders,
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]:
```suggestion
    const doc = wrapper.find(Response);  
```

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 9]
```diff
@@ -83,3 +84,151 @@ test('should show different component tabs based on state', () => {
...
+            requestBody: null,
+            requestHeaders: [],
+            method: 'post',
+            responseHeaders: resultResponseHeaders,
+          }}
+        />
+      </IntlProvider>
+    );
+    const doc = wrapper.find('Response');  
+    doc.instance().setTab('metadata');
```
Comment (davidebianchi) [inline packages/api-explorer/__tests__/Response.test.jsx]: as above

File: packages/api-explorer/i18n/en.json [hunk 10]
```diff
@@ -1,4 +1,5 @@
 {
   "doc.definition": "Definition",
-  "tryit": "Try the API to see Results"
+  "tryit": "Try the API to see Results",
+  "header.noValue": "no-value"
```
Comment (davidebianchi) [inline packages/api-explorer/i18n/en.json]: no-value is not clear

File: packages/api-explorer/src/components/Response/ResponseMetadata.jsx [hunk 11]
```diff
@@ -77,7 +106,27 @@ function ResponseMetadata({ result, intl }) {
...
       <Meta label={intl.formatMessage({id: 'api.response.headers', defaultMessage: 'Response Headers'})}>
-        {result.responseHeaders.join('\n')}
+        <div style={style.headersContainer}>
+
+          {result.responseHeaders.map(header => {
+            return (
+              <div style={style.headerRow}>
+                <span style={style.headerName}>{header.name}</span>
+                { 
+                  header.value ? <span style={style.headerValue}>{header.value}</span>
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/Response/ResponseMetadata.jsx]: If the value is an empty string, leave the empty string

File: packages/api-explorer/i18n/en.json (hunk 10 above)
Comment (dibericky) [reply packages/api-explorer/i18n/en.json]: removed.

**Pull request merged**